client/coral-configure/components/ConfigureCommentStream.css

```diff
@@ -0,0 +1,37 @@
+.container {
```

Comment:
I agree. The idea of having the `default.css` was for customizing and creating themes. This idea still remains. But components need to have core styling such as positions, and vertical-aligns. The app must still work regardless of the `default.css`. That being said, shouldn't the work of `default.css` only be for theming and overriding purposes?